src/ui/colortableview.js

```diff
@@ -83,13 +88,37 @@ export default class ColorTableView extends View {
 		 */
 		this.removeButtonLabel = removeButtonLabel;
 
+		/**
+		 * The label for the section with document colors.
+		 *
+		 * @type {String}
+		 */
+		this.documentColorsLabel = documentColorsLabel;
```

Comment:
There's no need to expose this property. Document colors label will be used in the constructor and it is (for now) no longer needed afterwards.

Let's not expose it.